### ISSUE TITLE
"TypeError ... Converting circular structure to JSON ... property 'stateNode' closes the circle" in allure-cypress for some tests

### ISSUE THREAD

**Issue by alexeyhimself** [allure-framework/allure-js#1128]
**Describe the bug**
`npx cypress run` throws "TypeError ... Converting circular structure to JSON ... property 'stateNode' closes the circle" for some tests. Trying to figure out why, will update (tbd) sections as soon as possible.

**To Reproduce**
1. (tbd since Sep 4, 2024)
2. Run cypress tests:
`npx cypress run`

**Actual behavior**
```
17
TypeError
Converting circular structure to JSON
--> starting at object with constructor 'HTMLLIElement'
|     property '__reactFiber$5fkdlykf24c' -> object with constructor 'Mc'
--- property 'stateNode' closes the circle
node_modules/allure-cypress/dist/cjs/index.js:1:13384
> 1 | "use strict";var H=Object.defineProperty;var dt=Object.getOwnPropertyDescriptor;var mt=Object.getOwnPropertyNames;var yt=Object.prototype.hasOwnProperty;var gt=(e,t)=>{for(var s in t)H(e,s,
... (lots of lines removed) ...
{V(e)||at(e)}),Cypress.on("command:end",e=>{V(e)||ot()}),afterEach(_,ee),after(_,function(){ee(),R(this)}),ut())};Xt();
  2 | //# sourceMappingURL=index.js.map
  3 |
```

**Expected behavior**
No such errors happen

**Environment**
 - both on Mac and Windows
 - latest cypress, latest allure-cypress, allure-commandline
 
**Additional context**
(tbd since Sep 4, 2024)

**Comment by delatrie** [comment]
Hi, @alexeyhimself !
Please, check #1070, you're most probably facing the same issue. If removing Cypress commands from the test makes it pass, then that's it!

**Comment by delatrie** [comment]
The issue should be fixed in allure-cypress@3.0.0-beta.12. Could you please verify that?

**Comment by alexeyhimself** [comment]
I confirm that issue is fixed for this case with allure-cypress@3.0.0-beta.12.
Thank you! I close the issue
